pymatbridge/pymatbridge.py

```diff
@@ -169,6 +169,7 @@ def __init__(self, executable, socket_addr=None,
 
         self.context = None
         self.socket = None
+        self._initialized = False
```

Comment:
And not sure what role this plays, either. Wouldn't you rather just call `set_default_plot_size` in `start`?